prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -304,8 +308,9 @@ def get_network_adapter_data(self) -> Dict[str, int]:
             network_adapters: Optional[Dict[str, Any]] = self._verify_redfish_call(
                 self.redfish_obj, network_adapters_root_uri
             )
-            if not network_adapters:
+            if network_adapters is None:
```

Comment:
Ditto, what if network_adapters is an empty dict? 
1. If None and "[]" has different meaning  => you should handle the empty dict case, but you didn't
2. If None and "[]" are identical =>  you should not compare with "None" only. 

Either one, you have to change the code. The original coding is good to me.

Reply:
It slipped my mind about the empty dict case. Will handle this case and keep the original code which will take care of None (from `_verify_redfish_call`) and empty dict